.gitignore

```diff
@@ -79,6 +79,7 @@ web_modules/
 .env.test.local
 .env.production.local
 .env.local
+fed.env
```

Comment:
don't want to add as example? See: https://github.com/neurobagel/federation-api/issues/34